postgresql/endpoint_methods.go

```diff
@@ -0,0 +1,56 @@
+package postgresql
+
+import (
+	"github.com/lib/pq"
+
+	"github.com/benchttp/server/benchttp"
+)
+
+func (c *computedStatsService) ListMetadataByUserID(userID int) ([]*benchttp.Metadata, error) {
+	metadataList := make([]*benchttp.Metadata, 0)
+
+	stmt, err := c.db.Prepare("SELECT tag, finished_at FROM metadata WHERE user_id = $1 ORDER BY finished_at DESC")
+	if err != nil {
+		return []*benchttp.Metadata{}, ErrPreparingStmt
+	}
+	defer stmt.Close()
+
+	rows, err := stmt.Query(userID)
+	if err != nil {
+		return []*benchttp.Metadata{}, ErrExecutingPreparedStmt
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		metadata := new(benchttp.Metadata)
```

Comment:
[suggestion] (same about pointers)
```suggestion
		metadata := benchttp.Metadata{}
```